otter/test/convergence/test_service.py

```diff
@@ -1366,6 +1372,43 @@ def plan(*args, **kwargs):
             perform_sequence(self.get_seq() + sequence, self._invoke(plan)),
             ConvergenceIterationStatus.Stop())
 
+    def test_launch_stack_config(self):
+        lc = {'args': {'stack': {'stack_name': 'foo'}},
+              'type': 'launch_stack'}
+
+        self.manifest = {
+            'state': self.state,
+            'launchConfiguration': lc,
+        }
+
+        self.gsgi_result = (self.group, self.manifest)
+        self.gacd_runner = lambda i: {'stacks': []}
+
+        def plan(*args, **kwargs):
+            return []
+
+        executor = attr.assoc(launch_stack_executor,
+                              gather=intent_func('gacd'),
+                              plan=plan)
+
+        seq = [
+            parallel_sequence([]),
```

Comment:
Since these entries are kind of confusing I like having a comment here explaining why it's there (I know this isn't consistently done elsewhere in the file).  I think this is the here because we have no steps, right?